dispatcher/backend/src/tests/integration/routes/schedules/test_nautilus.py

```diff
@@ -0,0 +1,109 @@
+from collections import namedtuple
+from typing import List
+
+import pytest
+
+from common import constants
+
+
+def update_dict(dict: dict, key_path: str, new_value: any):
```

Comment:
Should we share this across tests? I assume this is not specific to Nautilus, but if it is stored here then we will not reuse it for sure.